packages/rspack-plugin-react-refresh/src/index.js

```diff
@@ -59,6 +63,12 @@ module.exports = class ReactRefreshRspackPlugin {
 			},
 			use: "builtin:react-refresh-loader"
 		});
+
+		const refreshPath = path.dirname(require.resolve("react-refresh"));
```

Comment:
we need to tell users to install react-refresh manully on docs, otherwise it will throw error to user, cc @ahabhgk